source/opt/ir_context.cpp

```diff
@@ -206,6 +223,17 @@ void spvtools::ir::IRContext::ForgetUses(Instruction* inst) {
       get_decoration_mgr()->RemoveDecoration(inst);
     }
   }
+  if (id_to_name_) {
```

Comment:
This code is almost identical to the code at line 109.   I'd like to see this pulled out as a helper method.